src/Localization/en/GraphAPICompute.json

```diff
@@ -0,0 +1,57 @@
+{
+    "GraphAPIDescription": "Provision a Graph GraphAPI Compute for your Azure Cosmos DB account.",
```

Comment:
add "DedicatedGateway": "Dedicated Gateway"

Reply:
dont need it anywhere